packages/datadog-plugin-redis/test/integration-test/client.spec.js

```diff
@@ -7,32 +7,37 @@ const {
   spawnPluginIntegrationTestProc
 } = require('../../../../integration-tests/helpers')
 const { assert } = require('chai')
+const { NODE_MAJOR } = require('../../../../version')
+
+// Error: Command failed: yarn add file:/tmp/701a143ad8465e61/dd-trace.tgz 'redis@^4'
+// error @redis/client@1.5.10: The engine "node" is incompatible with this module.
+// Expected version ">=18". Got "16.20.2" error Found incompatible module.
+const describe = NODE_MAJOR < 18 ? globalThis.describe.skip : globalThis.describe
 
 describe('esm', () => {
   let agent
   let proc
   let sandbox
+  withVersions('redis', 'redis', '>=4', version => {
```

Comment:
Should this not test against a different version range on older Node.js versions rather than skipping entirely?